README.md

```diff
@@ -1,18 +1,57 @@
 # thesis-travel-app
 
-Travel App
+# TRVL
 
-Team: OperationDeploy
-Members: Austin, Garrett, Jessa, Josh
+## Getting Started
 
-HOW TO START APP:
-run the following commands:
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
 
-front-end:
-production-> $ npm run build
-development-> $ npm run dev
+### Installing
 
-back-end:
-\$ npm run start
+A step by step series of examples that tell you how to get a development env running
 
-The development command is coupled with 'npm run start' so all you would need is to run: npm run dev
+1. Go the TRVL repository: thesis-travel-app.
+2. **_Fork_** the repo.
+3. Next, **_git clone_** your fork of the repo.
+4. Then, **_cd_** into the repo.
+5. Run the command **_npm install_** to download dependencies defined in the package.json file.
```

Comment:
make sure say *where* npm install needs to be run, in this app it will be root level.